model/riscv_insts_vext_utils.sail

```diff
@@ -976,8 +976,7 @@ function rsqrt7 (v, sub) = {
 
 val riscv_f16Rsqrte7 : (bits_rm, bits_H) -> (bits_fflags, bits_H) effect {rreg}
```

Comment:
Remove effect throughout.

Reply:
These lines are not part of this PR

Reply:
You are seeing them just because this PR was opened before my other PR removing the effects separately was merged.